doc/integrator/ngeo.rst

```diff
@@ -108,3 +108,23 @@ The map configuration will be here:
    If you previously had a CGXP application and want to keep your permalinks
    compatible, the resolutions should be the same as in the previous
    application.
+
+Dynamic.js view
+---------------
+
+To configure the ngeo constants with dynamic or configurable values we have the dynamic view.
+
+This view is configurable in the vars files in the section ``interfaces_config``.
+The sub section is the interface name, and after we have:
+
+* ``redirect_interface``: interface on witch one we can be redirected if we use the wrong type of device
+  (mobile/desktop).
+* ``do_redirect``: directly do the redirect.
+* ``constants``: Directly define an ``angular`` constant in the vars file.
+* ``routes``: key: constant name, value: name of the route witch one we want to have the URL.
```

Comment:
value description is not clear to me - I am not sure what we are trying to say here?

Reply:
Name of the route for which the URL is wanted. ?

Reply:
Yes, that sounds good